### DISCUSSION TITLE
a problem with my public server

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1592] [Q&A]
### Description

@bear101 i think i have to contact you to solve this problem: as other people made audio club public server or something, i want to make a public server called little Friends public server: i have pressed the publish publicly button in the teamtalk QT on this server, and i did change my server name to inclood #teamtalkpublish# as it requested me to do? and now the stats bot is coming to the server and going, but my server is not in public servers list: please help me and please verify my server: it's a nice public server.

### Application

- [ ] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by ChrisDuffley** [comment]
i'm not logging on at the moment, but I see it in the list here on my phone. Maybe reopen the client again? Also you may already know this but make sure that Include Unofficial Servers is checked in the Server List.

**Comment by amirmahdifard** [comment]
but i chekked it, and it's not there. how to make it in public list? i like it to be public

**Comment by amirmahdifard** [comment]
oh, now it's there, but i want to be a public

**Comment by amirmahdifard** [comment]
public servers has been removed: closing this
